README.md

```diff
@@ -95,7 +95,7 @@ As a system admin, you must create a webhook for each organization you want to r
    - **Content Type:** `application/json`
    - **Secret:** the webhook secret you copied previously.
 6. Select **Let me select individual events** for "Which events would you like to trigger this webhook?".
-7. Select the following events: `Branch or Tag creation`, `Branch or Tag deletion`, `Issue comments`, `Issues`, `Pull requests`, `Pull request review`, `Pull request review comments`, `Pushes`, `Stars`.
+7. Select the following events: `Branch or Tag creation`, `Branch or Tag deletion`, `Issue comments`, `Issues`, `Pull requests`, `Pull request review`, `Pull request review comments`, `Pushes`, `Stars`, `Releases`.
```

Comment:
The changelog should mention that GH admins need to update their hooks.

@ayusht2810 Do you have a suggestion where we could store that notice?

Reply:
@hanzei Should we manually update release notes to have that changelog or create a separate PR for this change to have a changelog for it?

Reply:
Release notes are essentially the changelog for plugin releases

Reply:
We have `Branch or Tag creation` already. Isn't that about the same as releases? I'm actually not sure if that tag creation feature works properly